app/services/automate_import_service.rb

```diff
@@ -31,9 +31,11 @@ def import_datastore(import_file_upload,
     reject_unrelated_namespaces(ae_import, domain_name_to_import_from, namespace_list)
     reject_unrelated_classes(ae_import, domain_name_to_import_from, class_list)
 
-    result = ae_import.import
-
-    File.delete("automate_temporary_zip.zip")
+    begin
+      result = ae_import.import
+    ensure
```

Comment:
This method could use some refactoring in that the zip file name should be used from a shared variable instead of separate strings to make sure it is consistent.

Suggested change (could be a follow PR):
Either
1. Set the `import_options` hash first and use the file name from there.  Also move the `ensure` to the end of the method along with the `File.delete` call.  Then the `begin` statement is no longer needed.

Or what I think is a better option:

2. Use the block form of `Tempfile.create` so a proper temp file is created and deleted at the end of the block without the caller having to explicitly delete it.  The name would be part of the object passed to the block.

If you decide to do the work in this PR I recommend a separate commit.